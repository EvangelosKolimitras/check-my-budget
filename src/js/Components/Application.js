import React from 'react'

export default class Application extends React.Component {
    render() {
        return(
           <div>
                <h2>Welcome to my MDA Boilerplate v1.0.0</h2>
                <h6> created by: <a href="http://e-kolimitras.com/"> Evangelos Kolimitras </a> </h6>
                <p>Contributions are mostly welcome and accepted</p>
           </div>
        )
    }
}
