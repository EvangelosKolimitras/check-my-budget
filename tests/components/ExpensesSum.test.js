import React from "react"
import { ExpenseSum } from "../../src/scripts/js/Components/ExpensesSum"
import { shallow, configure } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
configure({ adapter: new Adapter() })

test("should render sum of expenses for 1 input", () => {
    const wrapper = shallow(<ExpenseSum expensesCount={1} total={243} />)
    expect(wrapper).toMatchSnapshot()
})

test("should render sum of expenses for many inputs", () => {
    const wrapper = shallow(<ExpenseSum expensesCount={3} total={253243} />)
    expect(wrapper).toMatchSnapshot()
})
