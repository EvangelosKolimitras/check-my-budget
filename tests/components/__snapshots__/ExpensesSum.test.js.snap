// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render sum of expenses for 1 input 1`] = `ShallowWrapper {}`;

exports[`should render sum of expenses for many inputs 1`] = `ShallowWrapper {}`;
